app/services/folio_client.rb

```diff
@@ -176,16 +200,6 @@ def change_pin(token, new_pin)
     check_response(response, title: 'Assign pin', context: { user_id: patron_key })
   end
 
-  # Look up a patron by barcode and return a Patron object
-  # If 'patron_info' is false, don't run the full patron info GraphQL query
-  def find_patron_by_barcode(barcode, patron_info: true)
-    response = get_json('/users', params: { query: CqlQuery.new(barcode:).to_query })
-    user = response.dig('users', 0)
-    raise ActiveRecord::RecordNotFound, "User with barcode #{barcode} not found" unless user
-
-    patron_info ? Folio::Patron.find(user['id']) : Folio::Patron.new({ 'user' => user })
-  end
-
```

Comment:
Could we keep this, make `find_patron_by_university_id` do what it says, and then have a new method  that tries the other two methods in sequence? That's clearer to me than having `find_patron_by_university_id` the way it is in this PR

Reply:
(I can make this change if others agree)

Reply:
👍🏻

Reply:
That's fine with me too.. I was trying to leave the code in a "just take these few lines away later" state, but since it sounds like later may be... much later... clarifying it is good too 🤷‍♂️

Reply:
Updated to do this.